### ISSUE TITLE
Tablerates can be imported only once

### ISSUE THREAD

**Issue by JeroenVanLeusden** [tig-nl/postnl-magento2#110]
### Make sure you are using the *latest* version: https://tig.nl/postnl-magento-extensies/
Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running the latest version of the TIG PostNL Magento extension.

---

### What is the purpose of your *issue*?
- [ ] Feature request (request for a new functionality)
- [x] Bug report (encountered problems with the TIG PostNL Magento extension)
- [ ] Extension support request (request for adding support for a new extension)
- [ ] Other

---

### If your issue contains security sensitive information
- [x] This issue does not contain any security sensitive information

---

#### Steps to reproduce
- Export tablerates csv.
- Fill in rates and import csv.
- Add extra lines to the csv and upload again.

#### Expected result
Tablerates will be imported.

#### Actuel result
[POSTNL-0251] An error occurred while importing the table rates.

```
SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1-NL-0-*-package_value_with_d-15.0000-4.7500-0.0000' for key 'UNQ_F3036813B650ECE5B0358FB7FC3E6F21', query was: INSERT  INTO `tig_postnl_tablerate` (`website_id`,`dest_country_id`,`dest_region_id`,`dest_zip`,`condition_name`,`condition_value`,`price`) VALUES (?, ?, ?, ?, ?, ?, ?), (?, ?, ?, ?, ?, ?, ?)
```

On every import it tries to fully import the csv again without clearing the table first.

#### Workaround
Clear tablerates table before importing.

**Comment by tig-daanvandenbergh** [comment]
Thank you for the report, @JeroenVanLeusden.

I'll try to reproduce this locally and get back to you.

**Comment by JeroenVanLeusden** [comment]
An extra note; It's impossible to use the `Price vs. Destination` condition (value `package_value_with_discount`) because the length of `condition_name` field for table `tig_postnl_tablerate` is only 20 characters long, which will cut the condition into `package_value_with_d`.

**Comment by tig-jaspersmits** [comment]
Hi @JeroenVanLeusden ,

I've just tested this, but I'm unable to reproduce the problem. When I add a line and re-upload, the new file gets uploaded correctly without any problems.

**Comment by tig-jaspersmits** [comment]
I will close this issue due to not being able to reproduce the problem and not receiving a response for 3 months.

**Comment by igorwulff** [comment]
@tig-jaspersmits yes the file gets uploaded perfectly, but when you download it you only get a file with headers no further rows. Please check the comment more closely from JeroenVanLeusden. He is mentioning where the value 'package_value_with_discount' is stored, can't contain the full string.

**Comment by igorwulff** [comment]
I've created a new ticket: https://github.com/tig-nl/postnl-magento2/issues/140
